napari/_vispy/canvas.py

```diff
@@ -335,7 +338,7 @@ def _map_canvas2world(
         position_world = list(self.viewer.dims.point)
         for i, d in enumerate(self.viewer.dims.displayed):
             position_world[d] = position_world_slice[i]
-        return tuple(position_world)
+        return position_world[0], position_world[1]
```

Comment:
Shot in the dark, but as we get an index error, I think maybe this can have a len of 3.
```suggestion
        assert len(position_world) == 2
        return position_world[0], position_world[1]
```